### ISSUE TITLE
Logging.Abstractions should be 2.2.0 for netstandard2.0

### ISSUE THREAD

**Issue by nesc58** [dsbenghe/Novell.Directory.Ldap.NETStandard#133]
Hi,
I am unable to update to version 3.3.0.

The problem is that 3.3.0 requires `Microsoft.Extensions.Logging.Abstractions` in version `3.1.7`.

Using this in a `netcorapp2.2` application the installation is not longer possible.
`netcoreapp2.2` requires `Microsoft.Extensions.Logging.Abstractions` in version `2.2.0`.

**Comment by tibel** [comment]
Strange, I had no issues using logging abstractions 3.1 in netcoreapp2.1.
Can you describe this in more detail please.

**Comment by dsbenghe** [comment]
It is true. Unfortunately, I saw see the issue of references only after I merged to master and released 3.3.0.

**Comment by nesc58** [comment]
Here is the log entry

```
Severity	Code	Description	Project	File	Line	Suppression State
Warning	NU1608	Detected package version outside of dependency constraint: Microsoft.AspNetCore.App 2.2.0 requires Microsoft.Extensions.Logging.Abstractions (>= 2.2.0 && < 2.3.0) but version Microsoft.Extensions.Logging.Abstractions 3.1.7 was resolved.	Service	Service.csproj	1	
```
